core/primer.md

```diff
@@ -674,3 +674,50 @@ in the entity in order to be model compliant.
 
 Likewise, implementations of the server must validate the entire entity
 against the new model, not just a subset of the entity's attributes.
+
+# Why are `ancestor` attributes that point to self allowed?
+
+The [`ancestor` attribute](spec.md#ancestor-attribute) is used to build a
+hierarchy of Versions aiding in compatibility checking when the
+[`compatibility` attribute](spec.md#compatibility-attribute) is set.
+Generally speaking, implementations are not expected to allow users to
+explicitly create multiple roots for a single Version. However, the
+specification does not mandate this as there may be cases where it is useful
+to allow for multiple roots. In some cases it may even be unavoidable, for
+example when the `maxversions` attribute (See `groups.resources.maxversions`
+under [Resource Model](spec.md#registry-model)) is set to a value that forces
+pruning of the Version tree. In such cases, when deleting the oldest version,
+this could result in a new root being created when there are multiple
+decedents of the deleted Version.
+
+# What's the oldest Version of a Resource?
+
+The oldest Version of a Resource isn't necessarily the one with the oldest
+`createdat` timestamp, because the `ancestor` attribute takes precedence
+over the `createdat` attribute. This is because the ancestor tree more
+accurately describes an ordered lineage of the Versions than timestamps. In
+the case where `groups.resources.singleversionroot` is set to `true`, the oldest
+Version of a Resource (or the first one ordered alphabetically should
+multiple Versions with the same `createdat` timestamp exist) will always be
+the oldest one in practice as only a single root exists.
+To accurately determine the oldest Version of a Resource, independent of the
```

Comment:
I think you can remove the rest of this since it's part of the spec itself already